index.html

```diff
@@ -0,0 +1,225 @@
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item">JavaScript</li>
+            <li class="langage-item">HTML</li>
+            <li class="langage-item">CSS</li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+      <li class="card-item flex-center-culomn">
+        <img class="card-image" src="img/Snapshoot-Portfolio.png" alt="Portfolio Portfolio">
+        <div class="list-conatiner">
+          <h2>Tonic</h2>
+           <div class="list">
+            <span class="list-item item1">Canopy</span>
+            <span class="list-item">back end Dev</span>
+            <span class="list-item">2015</span>
+           </div>
+        </div>
+        <div class="text-container">
+          <p>
+            A daily selection of privately personalized reads; no accounts or sign-ups required.
+          </p>
+        </div>
+        <div class="languages-list-container">
+          <ul class="lang-container">
+            <li class="langage-item"><span>JavaScript</span></li>
+            <li class="langage-item"><span>HTML</span></li>
+            <li class="langage-item"><span>CSS</span></li>
+          </ul>
+        </div>
+        <button type="button" class="see-btn">See Project</button>
+      </li>
+
+    </ul>
+    </div>
+
+
+ 
+
+</section>
+
+<section >
+<div class="rgt-container">
+  <ul>
+    <li>
```

Comment:
- [ ] It seems that you forgot to implement this part below for About section, please Kindly implement it: 
![image](https://user-images.githubusercontent.com/71874425/159133144-61a0c00e-5ac6-4892-b779-6b3c4b72e0bd.png)